src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -157,123 +161,130 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
 
 # Start Che in offline mode
 docker run <other-properties> -v /tmp/offline:/data/backup eclipse/che:<version> start --offline
 ----
-
-The `--offline` parameter instructs the Che CLI to load all of the TAR files located in the folder mounted to `/data/backup`. These images will then be used instead of routing out to the Internet to check for DockerHub. The preboot sequence takes place before any CLI functions make use of Docker. The `eclipse/che start`, `eclipse/che download`, and `eclipse/che init` commands support `--offline` mode which triggers this preboot sequence.
++
+The `--offline` parameter instructs the Che CLI to load all the TAR files located in the folder mounted to the `/data/backup` file. These images are then used instead of routing out to the Internet to check for DockerHub. The preboot sequence takes place before any CLI functions make use of Docker. The `eclipse/che start`, `eclipse/che download`, and `eclipse/che init` commands support the `--offline` mode that triggers this preboot sequence.
 
 [id="upgrade"]
-== Upgrade
+== Upgrading Che
 
-Upgrading Che is done by downloading a `eclipse/che-cli:<version>` that is newer than the version you currently have installed. You can run `eclipse/che-cli version` to see the list of available versions that you can upgrade to.
+To upgrade Che, download a newer version of it (`eclipse/che-cli:<version>`). For a list of available versions that you can upgrade to, run the `eclipse/che-cli version` command.
 
-For example, if you have 6.0.0 installed and want to upgrade to 6.0.1, then:
+*Example*
 
-----
-# Get the new version of Che
-docker pull eclipse/che-cli:6.0.0
+If you have 6.0.0 installed and want to upgrade to 6.0.1:
 
-# You now have two eclipse/che-cli images (one for each version)
-# Perform an upgrade - use the new image to upgrade old installation
-docker run <volume-mounts> eclipse/che-cli:6.0.1 upgrade
-----
+. Get the new version of Che: `docker pull eclipse/che-cli:6.0.0`.
++
+You now have two eclipse/che-cli images (one for each version).
++
+. Use the new image to upgrade the old installation: `docker run <volume-mounts> eclipse/che-cli:6.0.1 upgrade`.
++
+The upgrade command has numerous checks to prevent you from upgrading Che if the new image and the old one are not compatible. For the upgrade procedure to advance, the CLI image must be newer than the one in the `/instance/che.ver` file.
 
-The upgrade command has numerous checks to prevent you from upgrading Che if the new image and the old version are not compatible. In order for the upgrade procedure to advance, the CLI image must be newer that the version in `/instance/che.ver`.
+To upgrade Che, take the following steps:
 
-The upgrade process:
+. Perform a version compatibility check.
+. Download the new Docker image needed to run the new version of Che.
+. If it is running, stop Che.
+. Trigger a maintenance window.
+. Take a backup of your installation.
+. Initialize the new version.
```

Comment:
how?

Reply:
Better to include a link to that page.

Reply:
Added the link and also commands to the other steps.

. Perform a version compatibility check.
. Download the new Docker image needed to run the new version of Che.
. Stop Che using the `stop` command.
. Trigger a maintenance window.
. Back up your installation using the `backup` command.
. Initialize the new version using the `init` command 
. Start Che using the `start` command. (For a list of these commands, see https://www.eclipse.org/che/docs/docker-cli.html[CLI Syntax and Commands].)